SPECS/subversion/subversion.spec

```diff
@@ -0,0 +1,55 @@
+Summary:    	The Apache HTTP Server
```

Comment:
I will ask Siju to upload two files ,subversion and mesos to the bintray.

Thanks,
Sarah

From: Alexey Makhalov <notifications@github.com<mailto:notifications@github.com>>
Reply-To: vmware/photon <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, July 16, 2015 at 1:53 PM
To: vmware/photon <photon@noreply.github.com<mailto:photon@noreply.github.com>>
Cc: Sarah Choi <sarahc@vmware.com<mailto:sarahc@vmware.com>>
Subject: Re: [photon] Update for mesos (#186)


In SPECS/subversion/subversion.spec<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_photon_pull_186-23discussion-5Fr34836568&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=c4OdIZRLJOeaKdlMBB1ObQ5T0xt8wtM67EzY06DVEp8&m=rFflXHc0lRM68773GuBYiV0iFCL7VO3yaewob2b_xTM&s=777eBYKvY4HxaF1cDVP5Ubq9O4b6s6ioEpcR3tu0q7o&e=>:

> @@ -0,0 +1,55 @@
> +Summary:     The Apache HTTP Server


Subversion file is not found on bintray! From: Touseef Liaqat <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> Reply-To: vmware/photon <reply@reply.github.com<mailto:reply@reply.github.com><mailto:reply@reply.github.com>> Date: Tuesday, July 14, 2015 at 1:01 PM To: vmware/photon <photon@noreply.github.com<mailto:photon@noreply.github.com><mailto:photon@noreply.github.com>> Subject: Re: [photon] Update for mesos (#186<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_photon_pull_186&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=c4OdIZRLJOeaKdlMBB1ObQ5T0xt8wtM67EzY06DVEp8&m=rFflXHc0lRM68773GuBYiV0iFCL7VO3yaewob2b_xTM&s=lE4DuKB8_janW56ak0ghI4euiFbueN6a_I6ZgKVCXSk&e=>)
In SPECS/subversion/subversion.spec<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_photon_pull_186-23discussion-5Fr34612367&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=-haYzmx278tKb9Xx1ELbeFxCo1wTihwGUppL0Nz2QtE&m=K2wti2NvB_2gWYc-CbgmjLPiZw1-bT2h1KW2QpG91bA&s=J-obmAj_YwTsPyerqu-hszsC7TTmMq4tO6UggH746jE&e=>:
@@ -0,0 +1,55 @@ +Summary: The Apache HTTP Server
Fix Summary. - Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_photon_pull_186_files-23r34612367&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=-haYzmx278tKb9Xx1ELbeFxCo1wTihwGUppL0Nz2QtE&m=K2wti2NvB_2gWYc-CbgmjLPiZw1-bT2h1KW2QpG91bA&s=BIv4rWYOhDWGf587XZfKGall0Kpe5IozXo4WQz9ETyE&e=>.

-
Reply to this email directly or view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_vmware_photon_pull_186_files-23r34836568&d=BQMCaQ&c=Sqcl0Ez6M0X8aeM67LKIiDJAXVeAw-YihVMNtXt-uEs&r=c4OdIZRLJOeaKdlMBB1ObQ5T0xt8wtM67EzY06DVEp8&m=rFflXHc0lRM68773GuBYiV0iFCL7VO3yaewob2b_xTM&s=UkEFkQwbQrGBbm9I29hTrT_AxuKP1DtuG_5Fp7_rm8w&e=>.